src/main/java/net/skinsrestorer/shared/storage/Config.java

```diff
@@ -80,8 +85,13 @@ public static void load(String path, InputStream is) {
         DEFAULT_SKINS = config.getStringList("DefaultSkins.Names", ".skin");
         DISABLED_SKINS_ENABLED = config.getBoolean("DisabledSkins.Enabled", DISABLED_SKINS_ENABLED);
         DISABLED_SKINS = config.getStringList("DisabledSkins.Names");
+        SKINSGUI_ENABLED = config.getBoolean("SKINSGUI.Command.Enabled", SKINSGUI_ENABLED);
+        SKINSGUI_REGISTER_CMD = config.getBoolean("SKINSGUI.Command.RegisterGUICommand", SKINSGUI_REGISTER_CMD);
+        SKINSGUI_DISSABLED_MSG = config.getString("SKINSGUI.Command.DisableMessage", SKINSGUI_DISSABLED_MSG);
+        SKINSGUI_EXCLUDE_URL = config.getBoolean("SKINSGUI.ExcludeUrlSkins", SKINSGUI_EXCLUDE_URL);
+        SKINSGUI_SKULLAMOUNT = config.getInt("SKINSGUI.SkullAmount", SKINSGUI_SKULLAMOUNT);
         CUSTOM_GUI_ENABLED = config.getBoolean("CustomGUI.Enabled", CUSTOM_GUI_ENABLED);
-        CUSTOM_GUI_ONLY = config.getBoolean("CustomGUI.ShowOnlyCustomGUI", CUSTOM_GUI_ONLY);
+        CUSTOM_GUI_ONLY = config.getBoolean("CustomGUI.ShowOnlyCustomNames", CUSTOM_GUI_ONLY);
         CUSTOM_GUI_SKINS = config.getStringList("CustomGUI.Names");
```

Comment:
Better remove it before you forget it. It's already gone in the config yml file.